VideoLocker/src/androidTest/java/org/edx/mobile/test/http/SegmentTests.java

```diff
@@ -25,10 +28,13 @@ private void initTracker() {
         if (segment == null) {
             // create mocked instance of SegmentTracker
             tracker = Mockito.mock(ISegmentTracker.class);
```

Comment:
You might be interested in how I set this up on iOS: See https://github.com/edx/edx-app-ios/blob/master/edXVideoLockerTests/OEXMockAnalyticsTracker.h and https://github.com/edx/edx-app-ios/blob/master/edXVideoLockerTests/OEXMockAnalyticsTracker.m

and here's an example test using it: https://github.com/edx/edx-app-ios/blob/master/edXVideoLockerTests/OEXAnalyticsTests.m

The idea is that there is a simple interface that any analytics tracker can implement. Then I created a test analytics tracker that records what events fire. Tests can then just examine the events as data.